tests/testthat/test-as_flex_table.R

```diff
@@ -271,6 +271,63 @@ test_that("as_flex_table passes table footnotes & footnote abbreviations correct
     c(fn1[2], fn2[2]), # correct labels
     c("another new footnote", "replace old footnote")
   )
+
+  # testing one footnote passed to multiple columns and rows, addresses issue #2062
+  expect_silent(
+    my_tbl_summary |>
+      modify_footnote(stat_0 = NA) |>
+      modify_table_styling(
+        columns = c(label, stat_0),
+        rows = (variable %in% "trt") & (row_type == "level"),
+        footnote = "my footnote"
+      ) |>
+      as_flex_table()
```

Comment:
@Melkiades could you add a test here for the values of the footnotes too? thank you!

Reply:
done ;)